daemon/container_operations_unix.go

```diff
@@ -615,7 +615,7 @@ func (daemon *Daemon) connectToNetwork(container *container.Container, idOrName
 
 	ep, err := container.GetEndpointInNetwork(n)
 	if err == nil {
-		return fmt.Errorf("container already connected to network %s", idOrName)
+		return fmt.Errorf("a container with same name already connected to network %s", idOrName)
```

Comment:
perhaps:

`Conflict. A container with this name is already connected to network %s`

which would be consistent with the error we're using for conflicting container names;

    Error response from daemon: Conflict. The name "foobar" is already in use by container 98eb243f5cce......

Should we also include the name of the container? Not sure if that's easy to do

/cc @aboch